HopsanGUI/HcomHandler.cpp

```diff
@@ -5167,7 +5175,7 @@ void HcomHandler::executeBodeCommand(const QString cmd)
     int fMax = 500;
     if(args.size() > 2)
     {
-        fMax = args[2].toInt(); //!< @todo parse check needed
+        fMax = args[2].toInt()/(2*M_PI); //!< @todo parse check needed
```

Comment:
Is the default value for fMax 500 reasonable now that its rad/s ?

Reply:
Solved by multiplying with 2pi.